modules/search/instant-search/components/search-results.jsx

```diff
@@ -94,16 +78,33 @@ class SearchResults extends Component {
 						{ sprintf( __( 'No results for "%s"', 'jetpack' ), query ) }
 					</p>
 				) }
-				<ol className="jetpack-instant-search__search-results-list">
-					{ results.map( this.renderResult ) }
-				</ol>
-				{ this.props.hasNextPage && (
-					<ScrollButton
-						enableLoadOnScroll={ this.props.enableLoadOnScroll }
-						isLoading={ this.props.isLoading }
-						onLoadNextPage={ this.props.onLoadNextPage }
-					/>
+				{ results && (
+					<ol
+						className={ `jetpack-instant-search__search-results jetpack-instant-search__is-format-${ this
```

Comment:
This shares the same class name as the enclosing `main` on line 68. Is this intentional? I'd intentionally split out the ordered list into `.jetpack-instant-search__search-results-list` to avoid this. 

Also, I just realized we could remove `role="list"` from `main` in line 87...

Reply:
Not intentional - the rebase after the a11y PR merged was "fun" and this is an unintended side effect. I'll fix it up.

Reply:
Also removed `role="list"` from `<main>` as suggested 👍